pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -1,3 +1,42 @@
+let
+
+  #
+  # This is a list of pnames of packages that have failed to splice.
+  # Every one of these is an unfixed bug.  Please fix them instead
+  # of taking shameful behavior, you scoundrel!
+  #
+  list-of-shame =
+    builtins.listToAttrs
+      (builtins.map
+        (name: { inherit name; value = true; })
+        [
+          # shame on you, python
+          "wrap-python-hook"
+          "python-imports-check-hook.sh"
+          "python-namespaces-hook.sh"
+          "wrap-python-hook"
+          "pytest-check-hook"
+          "python-remove-tests-dir-hook"
+          "python-catch-conflicts-hook"
+          "python-remove-bin-bytecode-hook"
+          "python-output-dist-hook"
+          "python-remove-tests-dir-hook"
+          "wrap-python-hook"
+          "setuptools-setup-hook"
+          "pypa-build-hook.sh"
+          "pypa-install-hook"
+          "python3-3.11.5-env"
+          "setuptools-check-hook"
+          "unittest-check-hook"
+
+          # shame on you, llvm!
+          "llvm"
+          "compiler-rt"
+          "libcxx"
+          "libcxxabi"
```

Comment:
the llvm sets use the normal `makeScope` and that's why there's no `__spliced`'s  in the sets own `callPackage`

I'm little by little working on making the sets better https://github.com/NixOS/nixpkgs/issues/253671
once I dedupe more stuff it'll be much easier to dedupe the `llvm/*/default.nix`'es

Nice list name.